PendingReleaseNotes.md

```diff
@@ -20,8 +20,10 @@
 - PriorityClassNames can now be added to the Rook/Ceph components to influence the scheduler's pod preemption.
   - mgr/mon/osd/rbdmirror: [priority class names configuration settings](Documentation/ceph-cluster-crd.md#priority-class-names-configuration-settings)
   - filesystem: [metadata server settings](Documentation/ceph-filesystem-crd.md#metadata-server-settings)
-  - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)    
+  - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)
   - nfs: [samples](Documentation/ceph-nfs-crd.md#samples)
+- New CR property available in the Operator: `ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE`(5 seconds by default). Represents the time to wait until the node controller move Rook pods to other nodes after detecting a unreachable node. Pods affected by this setting are: mgr, rbd, mds, rgw, PVC based mons and osds, and ceph toolbox. The value used in this variable replaces the default value of 300 seconds added automatically by k8 as Pod Toleration for `node.kubernetes.io/unreachable`.
+Now the total amount of time to reschedule Rook pods in healthy nodes before detecting a `not ready node` condition will be the sum of `node-monitor-grace-period` (40 seconds by default) and `UNREACHABLE_NODE_TOLERATION_SECONDS` (5 seconds by default)
```

Comment:
Please indicate that `node-monitor-grace-period` is a kubernetes setting and what k8s component it is a setting for.

Reply:
Thanks. Better in this way :-)